guide/coaching-company.md

```diff
@@ -135,16 +135,13 @@ could do with your support.
 
 <h2 id="q6">How to become a Coaching Company</h2>
 
-First, register yourself and your coaches here: <a href="http://teams.railsgirlssummerofcode.org">teams.railsgirlssummerofcode.org</a>
+* First, look for a team in your city; this might require reaching out to your local <a href="http://railsgirls.com/">Rails Girls chapter</a> and to development communities such as your local Ruby groups to let them know that you're willing to host a team.
```

Comment:
Rails Girls "chapter"? Maybe organisational team or something like that would be better?